x/stakeibc/keeper/update_redemption_rates_test.go

```diff
@@ -0,0 +1,272 @@
+package keeper_test
+
+import (
+	// "fmt"
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	_ "github.com/stretchr/testify/suite"
+
+	recordtypes "github.com/Stride-Labs/stride/x/records/types"
+
+	// "github.com/Stride-Labs/stride/x/stakeibc/types"
+	stakeibc "github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+type UpdateRedemptionRatesTestCase struct {
+	host_zone          stakeibc.HostZone
+	redemption_rate_t0 sdk.Dec
+	staked_bal         uint64
+	undelegated_bal    uint64
+	just_deposited_bal uint64
+	st_supply          uint64
+}
+
+func (s *KeeperTestSuite) SetupUpdateRedemptionRates(
+	STAKED_BAL uint64,
+	UNDELEGATED_BAL uint64,
+	JUST_DEPOSITED_BAL uint64,
+	ST_SUPPLY uint64,
+	REDEMPTION_RATE_T0 sdk.Dec,
```

Comment:
looks like this parameter only ever equals 1 in these tests. If that's the case, I would just hard code it in this setup function and remove this parameter from the function definition

Reply:
good catch, just added a test that starts with a redemption rate that's not 1